src/unsafe-keyword.md

```diff
@@ -0,0 +1,79 @@
+# The `unsafe` keyword
+
+The `unsafe` keyword in Rust can occur in several different context: unsafe
```

Comment:
We try to avoid "in Rust" since everything in the reference is about Rust. I also think this seems like it should be plural.

```suggestion
The `unsafe` keyword can occur in several different contexts: unsafe
```